tests/functional/directives.ts

```diff
@@ -378,6 +397,18 @@ describe("Directives", () => {
       });
     });
 
+    describe("SubInputType", () => {
```

Comment:
it shouldn't be own describe block with type name, but rather just a `it` test case like "adds inherited field directives to input type fields while extending input type class"